docs/root/configuration/listeners/network_filters/kafka_mesh_filter.rst

```diff
@@ -0,0 +1,96 @@
+.. _config_network_filters_kafka_mesh:
+
+Kafka Mesh filter
+===================
+
+The Apache Kafka mesh filter provides a facade for `Apache Kafka <https://kafka.apache.org/>`_
+producers. Produce requests sent to this filter insance can be forwarded to one of multiple
+clusters, depending on configured forwarding rules. Corresponding message versions from
+Kafka 2.4.0 are supported.
+
+.. attention::
```

Comment:
Can you also add an attention/warning here that this is only available on contrib builds with a link? See what we do for squash and some of the other contrib filters and please unify as needed, thanks.

Reply:
done - comments and missing link added; same for kafka-broker